calico-cloud/index.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Calico Enterprise
+title: Calico Cloud
 description: Learn about Calico Enterprise
```

Comment:
Probably this should be "Learn about Calico Cloud."

Reply:
Fixed